twisted/spread/pb.py

```diff
@@ -981,14 +1005,16 @@ def proto_error(self, requestID, fail):
     def sendDecRef(self, objectID):
         """(internal) Send a DECREF directive.
         """
-        self.sendCall("decref", objectID)
+        self.sendCall(b"decref", objectID)
 
     def proto_decref(self, objectID):
         """(internal) Decrement the reference count of an object.
 
         If the reference count is zero, it will free the reference to this
         object.
         """
+        if isinstance(objectID, unicode):
+            objectID = objectID.encode('utf8')
         refs = self.localObjects[objectID].decref()
```

Comment:
I saw there are many other places in which we convert the key for localObjects. With that in mind I think that a dict like class is needed.

With manual conversion we might forget to convert at some place and then we end up with errors.